app/javascript/react/components/index/AnimalTile.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const AnimalTile = props => {
+  return(
+    <div>
+      <p>
+        {props.name}
+      </p>
+    </div>
+  )
+}
```

Comment:
Not sure if this needs to be inside both a div and a p tag. if you want the div for the sake of adding margin, I would add that margin with some CSS